File: app/src/main/java/com/hanix/randomgame/view/tab/body/BodyActivity.java
package com.hanix.randomgame.view.tab.body;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

import com.hanix.randomgame.R;
import com.hanix.randomgame.view.event.OnSingleClickListener;

public class BodyActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_body);
    }

    /**
     * findViewById 및 OnClickListener 설정
     */
    private void init() {

    }

    /**
     * onClickListener -> 중복 클릭 방지를 위해 OnSingleClickListener 사용
     */
    OnSingleClickListener bodyClick = new OnSingleClickListener() {
        @Override
        public void onSingleClick(View v) {

        }
    };
}

File: app/src/main/java/com/hanix/randomgame/view/tab/body/BodyActivity.kt
package com.hanix.randomgame.view.tab.body

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.hanix.randomgame.R
import com.hanix.randomgame.view.event.OnSingleClickListener

class BodyActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_body)
    }

    /**
     * findViewById 및 OnClickListener 설정
     */
    private fun init() {}

    /**
     * onClickListener -> 중복 클릭 방지를 위해 OnSingleClickListener 사용
     */
    var bodyClick: OnSingleClickListener = object : OnSingleClickListener() {
        override fun onSingleClick(v: View) {}
    }
}